packages/provider/common.yml

```diff
@@ -0,0 +1,14 @@
+applications:
+  - crossplane
+  - appcat
+
+parameters:
+  components:
+    crossplane:
+      url: https://github.com/projectsyn/component-crossplane.git
+      version: v2.2.0
+
+    appcat:
+      url: https://github.com/vshn/component-appcat.git
+      version: provider-cloudscale
```

Comment:
Unfortunately, we can't use something like `HEAD` to reference a current local state.

Reply:
Currently components are fetched from the provided URL by `package compile`, so local state can't be referenced.

Reply:
Yes, I'm aware and that's fine for now.
Let's deal with that problem separately.